src/app/pagination/pagination.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="content-view-pf-pagination list-view-pf-pagination clearfix">
+  <div class="form-group">
+    <div class="padding-right-10">
+      <div class="btn-group dropdown" dropdown>
+        <button class="btn btn-default dropdown-toggle" dropdownToggle>
+        {{config.pageSize}}
+        <span class="caret"></span>  
+      </button>
+        <ul class="dropdown-menu" *dropdownMenu>
+          <li *ngFor="let increment of config.pageSizeIncrements" [ngClass]="{'selected': increment === config.pageSize}" class="display-length-increment">
+            <a role="menuitem" (click)="onPageSizeChange(increment)">{{increment}}</a></li>
+        </ul>
+      </div>
+    </div>
+    <span class="per-page-label">per page</span>
```

Comment:
For accessibility, we should add 'for='pageSizeMenu' to the 'per page' label span element. In this case, I'm thinking #pageSizeMenu would be added to the button element.